src/Cassandra/Metrics/Providers/Null/NullDriverCounter.cs

```diff
@@ -0,0 +1,37 @@
+//
+//       Copyright (C) 2019 DataStax Inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+
+using Cassandra.Metrics.Abstractions;
+
+namespace Cassandra.Metrics.Providers.Null
+{
+    internal class NullDriverCounter : IDriverCounter
```

Comment:
Hmm I named these `Null`  as a reference to the null object design pattern but if you think this might confuse other devs that look at the code I'll change it to `Empty` which should be clearer.

Reply:
bah, makes sense to continue using `Null`.